hugegraph-it/hg-it-minicluster/src/main/java/org/apache/hugegraph/it/node/StoreNodeWrapper.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hugegraph.it.node;
+
+import static org.apache.hugegraph.it.base.ClusterConstant.CONF_DIR;
+import static org.apache.hugegraph.it.base.ClusterConstant.JAVA_CMD;
+import static org.apache.hugegraph.it.base.ClusterConstant.STORE_JAR_PREFIX;
+import static org.apache.hugegraph.it.base.ClusterConstant.getFileInDir;
+import static org.apache.hugegraph.it.base.ClusterConstant.isJava11OrHigher;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+
+public class StoreNodeWrapper extends AbstractNodeWrapper {
```

Comment:
the nodeWrap look like a proxy or endpoint for remote server, so maybe can name to xxxProxy direct.